buildchain/lint.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+
+
+"""Tasks for the linting.
```

Comment:
Not sure this should be part of the 'buildchain' (neither of the current `make` infrastructure, actually). This is all driven by `tox`, let's document the invocation as such.

Reply:
The problem with `tox` is that it seems that you can't have a "target" that run all the linter (do you confirm @jbertran?) so you would have to type `tox -e lint-shell` and `tox -e lint-yaml` (and soon `tox -e lint-python`, …).

Reply:
tox has no concept of hierarchy, only targets attached to an environment, so you can't specify a "meta-target" in such a way that, for example, the lint target would trigger the lint-yaml target _and_ the lint-shell target.

This is why in the current iteration, the meta-target is in the Makefile.

Reply:
`doit` being a task automation tool more than a build automation tool, I'm not disturbed by the `lint` target (`vagrantup` doesn't build anything as well if we go that way).

But if there is a clean `tox`-only solution I'm not against it :slightly_smiling_face:

Reply:
`vagrantup` doesn't build immediately, but does depend on `all-local`.

Reply:
> `doit vagrant` doesn't work in my setup because `_build/root/bootstrap.sh` is not executable.

Good catch!

There is a bug here (I should preserve the permissions/owners/… of the template when I render it).

(I mainly tested the build of the ISO, so I missed that one).

Reply:
Fixed

Reply:
I'll keep the target as-is for now (as it was present in the original Makefile and I'm porting what currently exists).

If we really want it, we can move it out in a later PR.